docs/manual.md

```diff
@@ -576,13 +576,19 @@ When debugging is enabled the kernel will use the same UART as U-Boot.
 
 ## ZCU102
 
-Initial support is available for the ZCU102.
+Initial support is available for the Xilinx ZCU102.
 
 **FIXME:** Additional documentation required here.
 
 The ZCU102 can run on a physical board or on an appropriate QEMU based emulator.
 
-An QEMU command line:
+Microkit produces a raw binary file, so when using U-Boot you must execute the image using:
+
+    => go 0x40000000
+
+Note that the loading address must be `0x40000000`.
```

Comment:
If you add this note, you should also say, why it must be this address? The note's wording feels a bit odd now to me.

Reply:
I can remove the note or say something like 'The image expects to be loaded at `0x40000000`'. I added the note because I wanted to make it clear that it's a fixed address. The reasoning for the fixed address is explained in the section about adding platform support, it will also be explained once https://github.com/seL4/microkit/issues/49 is resolved.

I think I will probably just remove the note to make it consistent with the TQMa8XQP documentation.

Thanks.